src/mame/namco/namcos23.cpp

```diff
@@ -4555,9 +4580,17 @@ ROM_START( aking )
 	ROM_REGION( 0x80000, "subcpu", 0 )  /* Hitachi H8/3002 MCU code */
 	ROM_LOAD16_WORD_SWAP( "ag1vera.ic3",   0x000000, 0x080000, CRC(266ac71c) SHA1(648a64adc0e4a2cefd71c31a6a71359b6c196430) )
 
-	ROM_REGION( 0x40000, "iocpu", 0 )   /* I/O board MB90F574 MCU code */
+	ROM_REGION( 0x40000, "iocpu", 0 )   /* I/O board HD643334 H8/3334 MCU code. Hacked firmware to change the JVS ID to 'FCA-1' */
+	ROM_LOAD( "asc3_io-c1.ic14", 0x000000, 0x020000, BAD_DUMP CRC(b6627a0e) SHA1(92343d527f4a62773c4495b9a04e7e16b9d6fb96) )
+
+#if 0
```

Comment:
I hate #if 0 code, because it always eventually drifts.  Please decide if you want the regions and no_dump in (and then remove the #if 0) or not (then remove the block, thanks :-)

Reply:
Decided on the former to remove the #if 0 code.